metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/sql/PartitionHolder.java

```diff
@@ -0,0 +1,56 @@
+/*
+ *  Copyright 2017 Netflix, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ */
+package com.netflix.metacat.connector.hive.sql;
+
+import com.netflix.metacat.common.server.connectors.model.PartitionInfo;
+import lombok.Data;
+import org.apache.hadoop.hive.metastore.api.Partition;
+
+/**
+ * A wrapper class to hold the Partition internal ids and the partition either as PartitionInfo or Partition.
+ * @author amajumdar
+ * @since 1.1.x
+ */
+@Data
+public class PartitionHolder {
+    private Long id;
+    private Long sdId;
+    private Long serdeId;
+    private PartitionInfo partitionInfo;
+    private Partition partition;
+
+    /**
+     * Constructor populating the ids and partitionInfo.
+     * @param id            partition id
+     * @param sdId          partition storage id
```

Comment:
Seems like it should be renamed to `partitionStorageId`. Or at least `storageId`

Reply:
Keeping it relevant to the table it corresponds to.